client.go

```diff
@@ -312,6 +336,25 @@ func (c *Client) ReadLink(p string) (string, error) {
 	}
 }
 
+// Symlink creates a symbolic link at 'newname', pointing at target 'oldname'
```

Comment:
This should be called Link, for compatibility with the os package, http://godoc.org/os#Link

You can do this in a followup if you like.